.github/workflows/cromwell-test-run.yml

```diff
@@ -1,12 +1,20 @@
 name: Cromwell Test Run
 
 on:
+  pull_request:
   workflow_dispatch:
 
 jobs:
   testrun:
+    if: ${{ github.event.pull_request.head.repo.fork == false }}
     runs-on: ubuntu-latest
     permissions: write-all
+    strategy:
+      matrix:
+        wdl:
+          - helloHostname/helloHostname.wdl
```

Comment:
i don't know the answer here, just asking. do you want to make it explicit for each file tested (so edit this and other yml file whenever a new wdl folder is added to tests) or maybe use a glob pattern? hmm although maybe that's not possible to do a glob thing

Reply:
I'm working on this rn

Reply:
Oooo, I like this idea, mind if I give it a shot?

On a related note, I added some automation in #32 that now requires each unit test to have the wdl script follow the same name as the directory and requires an `inputs.json` and `options.json`.

Reply:
Awesome, thanks @seankross !

Reply:
> Oooo, I like this idea, mind if I give it a shot?

Hey, sorry I didn't see this until now.